torchsynth/synth.py

```diff
@@ -214,15 +214,25 @@ def randomize(self, seed: Optional[int] = None):
         Randomize all parameters
         """
         if seed is not None:
-            # Profile to make sure this isn't too slow?
-            mt19937_gen = csprng.create_mt19937_generator(seed)
+            self.generator.manual_seed(seed)
```

Comment:
If we only use self.generator here, then maybe let's just put it here? Will it be a performance hit? Otherwise it's clearer

Reply:
negligible in terms of performance to switch out to here